### ISSUE TITLE
Vimeo url

### ISSUE THREAD

**Issue by RushabhJoshi** [felixgirault/multiplayer#1]
problem with vimeo url

https://vimeo.com/channels/staffpicks/126060304(not working)

manually parsed url to

https://vimeo.com/126060304

**Comment by felixgirault** [comment]
Hi,

What version of essence are you using ?
I tried the first URL and I got back information as usual.

What is your problem exactly ?

**Comment by RushabhJoshi** [comment]
I am using latest version of essence.
and problem is not with essence.

Problem is in multiplayer.
I want my video autoplay.

manualy i extended new class and i changed url of all video without any protocol so my https video worked as well

like
'youtube' => array(
                        ...
            'url' => 'http://www.youtube-nocookie.com/embed/%s',
            ...
            )

to

'youtube' => array(
            ...
            'url' => '//www.youtube-nocookie.com/embed/%s',
            ...
            )

secondly I manualy managed vimeo link 
in pattern of https://vimeo.com/{id}

**Comment by felixgirault** [comment]
Yes, I'm sorry I answered too fast, I'm used to get issues on essence :)
Thank you for the details, I'll see what I can do.
